src/signs/ui/display/SignImages.css

```diff
@@ -0,0 +1,4 @@
+.SignImages__signImage {
```

Comment:
Use lower case names for CSS classes: https://github.com/ElectronicBabylonianLiterature/ebl-frontend#htmlcss

If I remember correctly, we discussed this at some point.